src/server/types.ts

```diff
@@ -16,5 +18,6 @@ declare namespace ts.server {
         gc?(): void;
         trace?(s: string): void;
         require?(initialPath: string, moduleName: string): RequireResult;
+        importServicePlugin?(root: string, moduleName: string): Promise<ImportPluginResult>;

```

Comment:
@mjbvz and I discussed that [earlier](https://github.com/microsoft/TypeScript/pull/47377#discussion_r821190177). This isn't as generic as `import` because there is specific logic for importing a browser plugin inside that function.